dotty-tensorflow/src/main/scala/me.shadaj.scalapy.tensorflow/example/BidirectionalLSTMExample.scala

```diff
@@ -13,42 +16,39 @@ import scala.language.implicitConversions
 object BidirectionalLSTMExample extends Runnable {
 
   def run() = {
-    val tf = Modules.tensorflow
-    val np = Modules.numpy
     val keras1 = tf.keras
     val layers = keras1.layers
     val imdb = keras1.datasets.imdb
     val sequence = keras1.preprocessing.sequence
-    val max_features = 20000
+    val maxFeatures = 20000
     // cut texts after this number of words
-    // (among top max_features most common words)
+    // (among top maxFeatures most common words)
     val maxlen = 100
-    val batch_size = 32
+    val batchSize = 32
 
     println("Loading data...")
-    val ((x_train, y_train), (x_test, y_test)) = imdb.load_data(num_words = max_features)
-    println(s"${x_train.length} train sequences")
-    println(s"${x_test.length} test sequences")
+    val ((xTrain, yTrain), (xTest, yTest)) = imdb.loadData(numWords = Some(maxFeatures))
+    println(s"${xTrain.length} train sequences")
+    println(s"${xTest.length} test sequences")
 
     println("Pad sequences (samples x time)")
-    val x_train1 = sequence.pad_sequences(x_train, maxlen = maxlen).astype(np.float32)
-    val x_test1 = sequence.pad_sequences(x_test, maxlen = maxlen).astype(np.float32)
-    val y_train1 = y_train.astype(np.float32)
-    val y_test1 = y_test.astype(np.float32)
-    println(s"x_train shape: ${x_train1.shape}")
-    println(s"x_test shape: ${x_test1.shape}")
+    val xTrain1 = sequence.padSequences(xTrain, maxLen = Some(maxlen)).astype(np.float32)
+    val xTest1 = sequence.padSequences(xTest, maxLen = Some(maxlen)).astype(np.float32)
+    val yTrain1 = yTrain.astype(np.float32)
```

Comment:
There are no generic types implemented in lower layer (scala 2) for now, we picked `Float` as a universal type because it was convenient. 
But we can use `Numeric` in this layer and cast it to `Float` when passing to scala 2 layer.

Reply:
Made an issue, will take care of that later, It causes build problems